src/rust/bitbox02-rust/src/hww/api/backup.rs

```diff
@@ -89,6 +89,17 @@ pub async fn create(
     })
     .await?;
 
+    let is_vfat_formatted = bitbox02::sd::sdcard_vfat_formatted();
+    if !is_vfat_formatted {
+        confirm::confirm(&confirm::Params {
+            title: "SD card\nformatting needed",
+            body: "This will erase all\ndata on the SD card.",
+            ..Default::default()
+        })
+        .await?;
+        bitbox02::sd::sd_format();
```

Comment:
one reason to make sd_format() return a Result<> is so that the compiler can warn you if you don't handle the result :grin:

Reply:
True, I intended to ask this in the pr message but forgot. Thanks, now it also handles when format fails 👍